specification/streamanalytics/resource-manager/Microsoft.StreamAnalytics/preview/2017-04-01-preview/inputs.json

```diff
@@ -461,7 +471,10 @@
           "type": "string",
           "description": "Indicates the type of input data source containing stream data. Required on PUT (CreateOrReplace) requests."
         }
-      }
+      },
+      "required": [
```

Comment:
Why do you need this "required" annotation?

Reply:
I think I also see this for other "type" like properties across the files

Reply:
For fix SEMANTIC_VALIDATION_ERROR, it may be a new CI rule, so before code don't throw such error when checked in.
```
{
  code: 'SEMANTIC_VALIDATION_ERROR',
  id: 'OAV119',
  message: 'The spec .\\specification\\streamanalytics\\resource-manager\\Microsoft.StreamAnalytics\\preview\\2017-04-01-preview\\inputs.json has semantic validation errors.',
  innerErrors: [
    {
      code: 'OAV131',
      error: 'DISCRIMINATOR_NOT_REQUIRED',
      name: 'discriminator',
      params: [ 'type' ],
      message: 'discriminator must be a required property.',
      path: [ 'definitions,Serialization' ]
    }
  ],
  level: '\u001b[31merror\u001b[39m'
}
```